explorer/client/package.json

```diff
@@ -29,10 +30,15 @@
         "typescript": "^4.5.5"
     },
     "dependencies": {
+        "@faker-js/faker": "^7.3.0",
```

Comment:
How is this @faker-js module being used or should it be removed?

Reply:
I forgot to remove this, was using it to create a test UI. Will remove.